testing/config.go

```diff
@@ -427,6 +427,7 @@ var (
 		return str != "" && !strings.HasPrefix(str, "Status")
 	}, cmp.Ignore())
 
+	// Deprecated: use reconcilers.IgnoreAllUnexported instead
```

Comment:
**[praise]:** Thanks for this! Will schedule removal in next+1 version. https://github.com/search?q=SafeDeployDiff&type=code suggests that it's not used by public projects on Github at least.